routes/web.php

```diff
@@ -73,6 +75,21 @@
 Route::get('/albums/{year}', [GalleryController::class, 'showGallery'])->name('galerij_jaar');
 Route::get('/albums/{id}/{year}', [GalleryController::class, 'show'])->name('galerij_album');
 
+//News routes
+Route::resource('nieuws', NewsArticleController::class);
+Route::resource('nieuwsbrief', NewsLetterController::class);
+//Route::get('/nieuws', [NewsArticleController::class, 'sorting'])->name('sorting');
```

Comment:
Als deze niet nodig is, kun je 'm weghalen, neem ik aan?

Reply:
ja